yang/frr-nexthop.yang

```diff
@@ -25,40 +24,40 @@ module frr-nexthop {
   }
 
   typedef gateway-address {
-      type inet:ip-address;
+    type inet:ip-address;
   }
 
   typedef nexthop-type {
     type enumeration {
-      enum "ifindex" {
-	value 1;
-	description
-	  "Specific interface.";
+      enum ifindex {
+        value 1;
+        description
+          "Specific interface.";
```

Comment:
So here, when I look at this file, I see that the `enum` line is indented 6 spaces, then its dependents are indented with a tab, which resolves (in my editor) to 8 spaces. the `description` text is then indented with a tab and 2 spaces. isn't that the normal frr code style?

Reply:
@mjstapp  I have run the file through pyang indent command which formatted the entire file. for any fields in yang file indent is expected to be two spaces.  The `description` is not expected to be indented.

Reply:
my point is that it would be best to have consistency between the editors we use and whatever other tools may be used. otherwise, there will be ... pain. in this case, and others like it, the formatting was correct, in frr terms - I think. can the pyang tool be told to respect 8-space tabs? or do you think we have to have a yang-specific style adjustment for every developer's editor in order to accomodate pyang?

Reply:
The YANG guidelines RFC doesn't mention anything about tabs vs spaces, but I did a quick inspection and 99% of the IETF YANG modules are using solely spaces for indentation. So I think we should do that as well.

> or do you think we have to have a yang-specific style adjustment for every developer's editor in order to accomodate pyang?

I'd say yes. YANG and C are different languages, so it's a natural thing they need different editor settings.